tests/test_main.py

```diff
@@ -347,9 +351,9 @@ def test_order_service_create_order_without_date(create_order_data, tickets):
 
 def test_order_service_create_order_with_date(create_order_data, tickets):
     create_order(tickets=tickets, username="user_1", date="2020-11-10 14:40")
```

Comment:
The `create_order` function is called with a string for the `date` parameter. Ensure that the `create_order` function can handle string dates or convert them to `datetime` objects before use.